### PR TITLE
feat: TT-134 job list pagination

### PR DESCRIPTION
**Pull request by safeamiiir**
Add pagination for the job list

### CODE REVIEW

File: aqueductcore/backend/routers/graphql/resolvers/task_resolver.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ async def get_tasks(
         order_by_creation_date=True,
     )
     task_nodes = [task_model_to_node(value=item) for item in tasks][offset : offset + limit]
-    return Tasks(tasks_data=task_nodes, total_tasks_count=len(task_nodes))
+    return Tasks(tasks_data=task_nodes, total_tasks_count=len(tasks))
```
Comment (str-anger) [inline aqueductcore/backend/routers/graphql/resolvers/task_resolver.py:41]: I assume this is a bugfix

File: aqueductcore/frontend/src/pages/JobHistoryPage/index.tsx [hunk 2]
```diff
@@ -7,10 +7,10 @@ import JobExtensionStatus from "components/molecules/JobListTableCells/JobExtens
 import JobExperimentName from "components/molecules/JobListTableCells/JobExperimentName";
 import { drawerTopOffset, mainPadding } from "components/templates/drawerLayout";
 import { dateFormatter, jobHistoryTableFormatter } from "helper/formatters";
-import { experimentRecordsRowsPerPageOptions } from "constants/constants";
 import { useGetAllTasks } from "API/graphql/queries/tasks/getAllTasks";
 import { JobDataType, JobsListColumnsType } from "types/globalTypes";
 import { ExperimentData } from "types/graphql/__GENERATED__/graphql";
+import { jobListRowsPerPageOptions } from "constants/constants";
```
Comment (str-anger) [inline aqueductcore/frontend/src/pages/JobHistoryPage/index.tsx:13]: You import something here but I feel like this something is not added? Or am I missing anything? How does this add pagination?

**Review by str-anger — CHANGES_REQUESTED** [review]
Can you please give some hints on what did you do here? :)

File: aqueductcore/frontend/src/pages/JobHistoryPage/index.tsx (hunk 2 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/pages/JobHistoryPage/index.tsx:13]: This is not adding pagination by itself, it's basically being corrected (it was mistakenly imported from a wrong constant)

File: aqueductcore/backend/routers/graphql/resolvers/task_resolver.py (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/backend/routers/graphql/resolvers/task_resolver.py:41]: It actually is a bugfix as i had added pagination halfway

**Review by samiralavi — APPROVED** [review]

**Review by str-anger — APPROVED** [review]

**Pull request merged**